Assets/Scripts/Combat/Combantants/Combatant.cs

```diff
@@ -178,7 +178,7 @@ public void ApplyShield(int shieldHealth)
         if (MaxShieldPoints == 0)
         {
             Shield.SetActive(true);
-            Shield.transform.position = transform.position;
+            Shield.transform.position = transform.position + new Vector3(0.0f, 2.0f, 0.0f);
```

Comment:
Make the ``new Vector3(0.0f, 2.0f, 0.0f);`` a variable and call it _Offset_. That way we can easily change it whenever we want.